db/migrate/20150724182915_add_subject_and_comment_template_for_email_settings.rb

```diff
@@ -0,0 +1,23 @@
+class AddSubjectAndCommentTemplateForEmailSettings < ActiveRecord::Migration
+  def change
+    add_column :email_settings, :comment_subject, :string
+    add_column :email_settings, :comment_template, :text
+    comment_subject = 'New comment has been posted for {eventtitle}'
+    comment_template_text = %q(Dear {name},
+
+User {comment_user} posted a new comment for event {eventtitle} of {conference} .
+
+" {comment_body} "
+
+To reply to this comment, please go to {comment_reply}
+
+Best wishes,
+{conference} Team
+)
+    EmailSettings.all.each do |email_setting|
```

Comment:
@raluka You only set the subject & template for existing email settings! That would mean when you create a new conference the related subject & template would be nil...

There're several ways to fix this:

1) Set default values in your migration
```
add_column :email_settings, :comment_subject, :string, :default => comment_subject
```

2) Set default values when we build the email settings for a new conference
https://github.com/openSUSE/osem/blob/master/app/models/conference.rb#L928

3) Add a feature to enter & change the the subject & template
https://github.com/openSUSE/osem/blob/master/app/views/admin/emails/index.html.haml

I'd go for the first or second implementation, I don't see much value in changing the comment email ...
But when we go for one of the first two solutions, the question is if we need to store the template & subject in the database at all?

@raluka Can you follow :)